web/pgadmin/tools/erd/__init__.py

```diff
@@ -386,6 +386,20 @@ def register_preferences(self):
             )
         )
 
+        self.preference.register(
+            'options',
+            'table_relations_depth',
+            gettext('Table Relations Depth'),
+            'integer',
+            -1,
+            category_label=PREF_LABEL_OPTIONS,
+            help_str=gettext(
+                'The maximum depth pgAdmin should traverse to find '
```

Comment:
The message should be "A maximum depth at which pgAdmin should search for related tables when generating an ERD. A value of -1 indicates that there is no limit."

Reply:
Using `A` is incorrect. There is only one input for depth, so it should be `The`

Reply:
@dpage Can you please suggest the strings here ?
Label: Table Relations Depth
Help: The maximum depth pgAdmin should traverse to find related tables when generating ERD for a table. Use -1 for no limit.